js/js-array-methods-2/js-array-methods-2_animal-strings/index.js

```diff
@@ -21,15 +21,22 @@ const animalStrings = [
   "rhino",
 ];
 
-const hippoExists = null;
+const hippoExists = animalStrings.includes("hippo");
 
 // Hint: You can pass the starting index as second parameter.
-const catStartingFromIndexFiveExists = null;
+const catStartingFromIndexFiveExists = animalStrings.includes("cat", 5);
```

Comment:
Interesting! Didn't know the second argument for the array.includes method was a starting index.